src/clients/python/library/tritonclient/utils/__init__.py

```diff
@@ -159,25 +159,27 @@ def triton_to_np_dtype(dtype):
 
 def serialize_byte_tensor(input_tensor):
     """
-        Serializes a bytes tensor into a flat numpy array of length prepend bytes.
-        Can pass bytes tensor as numpy array of bytes with dtype of np.bytes_,
-        numpy strings with dtype of np.str_ or python strings with dtype of np.object.
+    Serializes a bytes tensor into a flat numpy array of length prepended bytes.
+    Can pass bytes tensor as numpy array of bytes with dtype of np.bytes_,
```

Comment:
We want to recommend only np.object for both strings and byte blobs. We can also say "A numpy array of dtype np.str_ or np.bytes_ can also be used but are not recommended because numpy will remove trailing zeros at the end of the byte sequence for those types"